packages/gatsby/src/bootstrap/load-config/index.ts

```diff
@@ -94,7 +89,5 @@ export async function loadConfigAndPlugins({
 
   store.dispatch(internalActions.setSiteConfig(config))
 
-  const flattenedPlugins = await loadPlugins(config, siteDirectory)
-
-  return { config, flattenedPlugins }
+  return { config }
```

Comment:
What's the reason to return an object here? Do you want to expand it in the future?